ratelimit/ratelimit.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2023 Sauce Labs Inc. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package ratelimit
+
+import (
+	"github.com/saucelabs/forwarder/resource"
+	"golang.org/x/time/rate"
+)
+
+const defaultMaxBurstSize = 4 * 1024 * 1024 // Must be bigger than the biggest request.
+
+func newRateLimiter(bandwidth resource.SizeSuffix) *rate.Limiter {
+	// The rate limiter is configured to allow a burst of 1/64th of the bandwidth or defaultMaxBurstSize.
```

Comment:
Let's leave the original discussion and a pointer to rclone issue.
The point is that it does not allow high bandwidth if defaultMaxBurstSize is used.